src/_guides/language/language-tour.md

```diff
@@ -3649,17 +3649,20 @@ a possibly time-consuming operation
 (such as I/O),
 without waiting for that operation to complete.
 
-When you need to use a value represented by a Future,
-you have two options:
+The `async` and `await` keywords support asynchronous programming,
+letting you write asynchronous code that
+looks similar to synchronous code.
+
 
-* Use `async` and `await`
-* Use the [Future API](/guides/libraries/library-tour#future)
+<a id="await"></a>
+### Handling Futures
 
-Similarly, when you need to get values from a Stream,
+When you need to use a value represented by a **Future**,
 you have two options:
 
-* Use `async` and an _asynchronous for loop_ (`await for`)
-* Use the [Stream API](/guides/libraries/library-tour#stream)
+* Use `async` and `await` to handle the most common case:
+  waiting until the Future's value is available.
+* Use the [Future API](/guides/libraries/library-tour#future).
```

Comment:
I would have expected "Future API" to link to the Future API entry. I actually was a bit challenged to find a link to the Future API entry :).

How about making "Future API" link to the API entry and then mention that the reader can also consult the library tour section on Futures.

Reply:
I've modified the link text to clarify where it's going.